src/components/TalentPage/TalentPage.jsx

```diff
@@ -38,7 +39,7 @@ export function TalentPage() {
                         <p>{talent?.specialization}</p>
 
                         <div className={s.skills}>
-                            {talent.skills?.map((skill, talent) => (
+                            {talent.talents?.map((skill, talent) => (
```

Comment:
Same problem, better to use index than talent
```suggestion
                            {talent.talents?.map((skill, index) => (
```